docs/workflows.rst

```diff
@@ -44,29 +55,114 @@ warp to the MNI space.
+If enabled, several steps in the ``fmriprep`` pipeline are added or replaced.
+All surface preprocessing may be disabled with the ``--no-freesurfer`` flag.
+
+If FreeSurfer reconstruction is performed, the reconstructed subject is placed in
+``<output dir>/freesurfer/sub-<subject_label>/`` (see `FreeSurfer Derivatives`_).
+
+Surface reconstruction is performed in three phases.
+The first phase initializes the subject with T1- and T2-weighted (if available)
+structural images and performs basic reconstruction (``autorecon1``) with the
+exception of skull-stripping.
+For example, a subject with only one session with T1 and T2-weighted images
+would be processed by the following command::
+
+    $ recon-all -sd <output dir>/freesurfer -subjid sub-<subject_label> \
+        -i <bids-root>/sub-<subject_label>/anat/sub-<subject_label>_T1w.nii.gz \
+        -T2 <bids-root>/sub-<subject_label>/anat/sub-<subject_label>_T2w.nii.gz \
+        -autorecon1 \
+        -noskullstrip
+
+The second phase imports the brainmask calculated in the `T1w/T2w preprocessing`_
+sub-workflow.
+The final phase resumes reconstruction, using the T2-weighted image to assist
+in finding the pial surface, if available::
 
-.. image:: EPI_HMC.dot.png
+    $ recon-all -sd <output dir>/freesurfer -subjid sub-<subject_label> \
+        -all -T2pial
+
+Reconstructed white and pial surfaces are included in the report.
+
+.. figure:: _static/reconall.svg
     :scale: 100%
 
+    Surface reconstruction (FreeSurfer)
+
+If T1-weighted voxel sizes are less 1mm in all dimensions (rounding to nearest
+.1mm), `submillimeter reconstruction`_ is used.
+
+In order to bypass reconstruction in ``fmriprep``, place existing reconstructed
+subjects in ``<output dir>/freesurfer`` prior to the run.
+``fmriprep`` will perform any missing ``recon-all`` steps, but will not perform
+any steps whose outputs already exist.
+
+
+BOLD preprocessing
+------------------
+
+.. workflow::
+    :graph2use: orig
+    :simple_form: yes
+
+    from fmriprep.workflows.epi import bold_preprocessing
+    wf = bold_preprocessing("bold_preprocessing",
+                            metadata={"RepetitionTime": 2.0,
+                           "SliceTiming": [0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]},
+                 settings={'ants_nthreads': 1,
+                           'ignore':[],
+                                     'nthreads': 1,
+                                     'freesurfer': True,
+                                     'reportlets_dir': '.',
+                                     'output_dir': '.',
+                                     'bids_root': '.',
+                                     'biggest_epi_file_size_gb': 3,
+                                     'skull_strip_ants': True,
+                                     'skip_native': False,
+                                     'debug': False})
+
+Preprocessing of BOLD files is splict into multiple sub workflows decribed below.
+
+epi_hmc
```

Comment:
Can we replace the title in this fashion?:

```

.. epi_hmc :

:mod:`fmriprep.workflows.epi.epi_hmc` -- A head-motion correction (HMC) workflow
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

The first line is for generating an anchor that allows sphinx to use the `epi_hmc` identifier.